DMTN-248.tex

```diff
@@ -46,8 +46,244 @@
+``DELIGHT: Deep Learning Identification of Galaxy Hosts of Transients using Multiresolution Images" (Förster et al. 2022).
+
+The default 6$\times$6 arcsecond postage stamp size is large enough to contain the angular diameter (30 kpc) 
+of a Milky Way-like galaxy at a redshift of 0.5, which will be a fairly typical supernova host galaxy in the LSST data set.
+The fraction of all alerts that will be low-redshift transients with wide-area host galaxies will be $<$1\% \citedsp{DMTN-102}.
+
+Host association will already be included in the alert packet as part of the \texttt{DiaObject}, as described in \citeds{DMTN-151} 
+and in Table 3 of the DPDD, including potential nearby extended and low-redshift galaxies (see columns nearbyExtObj and nearbyLowzGal).
+
+It is expected that this type of host-galaxy identification could be done by accessing the template images 
+(which are not subject to the 80 hour embargo) via the image cutout service, via which large-area cutouts could be made.
+
+Although the context provided by larger postage stamps have also been shown to better identify \texttt{DiaSources} that are glints
+(rotating debris in low earth orbit that appears as a row of point sources), ways to identify and flag \texttt{DiaSources}
+as potential glints in the AP pipeline are underway, instead of leaving this identification solely to brokers.
+
+Furthermore, although the multi-resolution aspect of the proposal aims to provide a larger area without significantly
+adding to the size of the alert packet, it is currently unclear, e.g., how much extra header information is added, or whether the
+FITS would need multiple extensions.
+
+\textbf{Recommendation}: At this time, do not increase the size of postage stamps, but potentially reconsider in the future.
+
+
+\section{Packet compression}\label{sec:compression}
+
+\textbf{Proposal}: Compress alert packets to reduce their size.
+
+The application of gzip compression could further reduce the size of a full alert to 65 KB (80\%).
+This might help to avoid alert distribution bottlenecks, lower the cost for brokers' storage needs, and potentially enable 
+additional full streams in the future -- all of which could positively impact time-domain science with the LSST.
+
+However, the time and computational resources required to compress the alert packets needs to be considered.
+For example, gzip compression at 50 MB/s to compress 10000 alerts could take 10000 $\times$ 0.08 MB per alert, 50 MB/s, or 
+16 seconds, which is very significant in light of the 60 second alert distribution timescale. 
+
+Furthermore, compressing the alert packets forces brokers to then decompress the alerts on arrival, which would incur further delay.
+Science goals requiring very low-latency alerts distribution might be negatively impacted by compression.
+
+Compression would only be viable if an algorithm could provide a compression rate of 10 MB/s, requiring only a few seconds 
```

Comment:
This doesn't make sense: the compression speed should be *faster* than 50 MB/s per the logic above?

Reply:
Yeah good catch, I messed it up somehow -- deleted that line.